src/content/docs/en/editor-setup.mdx

```diff
@@ -32,7 +32,11 @@ import ReadMore from '~/components/ReadMore.astro';
 
 ## JetBrains IDEs
 
-Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).
+[Webstorm](https://www.jetbrains.com/webstorm/) is a JavaScript and TypeScript IDE that added support for the Astro Language Server in version 2024.2. This update brings features like syntax highlighting, code completion, and formatting.
+
+Install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. You can toggle the language server in `Settings | Languages & Frameworks | TypeScript | Astro`.
```

Comment:
Is this correct? The plugin and the language server are two different products.

Reply:
The plugin is required for the language server to be enabled. 

I've just confirmed on my own IDE that disabling the plugin removes the language server and the associated pane in the settings menu.

Reply:
@Princesseuh @sarah11918 Just wanted to follow up here.  Let me know if something is still unclear and I'll try to update the language again.

Reply:
Sorry, we've been trying to get the minor release out the door so other PRs have had to wait! This looks good, and if it needs updating, we can always make another PR!

Thank you again!

Reply:
No worries! Thanks!